web/src/engine/websites/Zebrack_e2e.ts

```diff
@@ -0,0 +1,25 @@
+﻿import { TestFixture, type Config } from '../../../test/WebsitesFixture';
+
+const config: Config = {
```

Comment:
Consider more test cases since there are various categories in the pathname
See: https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/CyComi_e2e.ts